calicoctl/calicoctl/commands/ipam/release.go

```diff
@@ -210,23 +210,28 @@ func releaseFromReports(ctx context.Context, c client.Interface, force bool, rep
 				if _, ok := notInUse[a.IP]; notInUse != nil && !ok {
 					continue
 				}
-				merged[a.IP] = struct{}{}
+				merged[a.IP] = a.Handle
 			}
 		}
 		notInUse = merged
 	}
 
-	ipsToRelease := []net.IP{}
-	for ip := range notInUse {
-		ipsToRelease = append(ipsToRelease, argutils.ValidateIP(ip))
+	ipsToRelease := []libipam.ReleaseOptions{}
+	for ip, handle := range notInUse {
+		// TODO: Should pass sequence number here as well.
```

Comment:
how come we aren't passing in the sequence number?

Reply:
Because I haven't written that code yet 😆 

Need to update the report generation code to include the seqNum in the report.